features/org.wso2.carbon.identity.organization.management.core.feature/resources/organization-mgt.xml

```diff
@@ -0,0 +1,68 @@
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Don't we need the `All Rights Reserved.` part? check the other places

Reply:
It's not required for public source code license